server/settings.py

```diff
@@ -61,8 +61,8 @@ def env(variable, fallback_value=None):
 SECRET_KEY = env('SECRET_KEY', '')
 
 # Highcharts Export Server - default settings
-ANALYTICS_ENABLE_SCHEDULED_REPORTS = strtobool(
-    env('ANALYTICS_ENABLE_SCHEDULED_REPORTS', 'true')
-)
-HIGHCHARTS_SERVER_HOST = env('HIGHCHARTS_SERVER_HOST', 'localhost')
-HIGHCHARTS_SERVER_PORT = env('HIGHCHARTS_SERVER_PORT', '6060')
+# ANALYTICS_ENABLE_SCHEDULED_REPORTS = strtobool(
+#     env('ANALYTICS_ENABLE_SCHEDULED_REPORTS', 'true')
+# )
+# HIGHCHARTS_SERVER_HOST = env('HIGHCHARTS_SERVER_HOST', 'localhost')
+# HIGHCHARTS_SERVER_PORT = env('HIGHCHARTS_SERVER_PORT', '6060')
```

Comment:
you could keep the settings but set the env vars to be empty/false via docker

Reply:
Leaving these lines but without env variables from docker causes a ```ValueError```, as an empty string is casted to an integer